src/imitation/algorithms/adversarial/airl.py

```diff
@@ -54,7 +55,13 @@ def __init__(
             reward_net=reward_net,
             **kwargs,
         )
-        if not hasattr(self.gen_algo.policy, "evaluate_actions"):
+        # AIRL needs a stochastic policy to compute the discriminator output.
+        # sac_policies.SACPolicy and policies.ActorCriticPolicy both have an actor
+        # to generate stochastic actions.
+        if not isinstance(
+            self.gen_algo.policy,
+            (sac_policies.SACPolicy, policies.ActorCriticPolicy),
```

Comment:
Consider defining a constant e.g. `STOCHASTIC_POLICIES` that defines this tuple. Then it's self-documenting (and you can abbreviate the comment).

Reply:
Good idea